TrueSDK/External/TCTrueSDK.m

```diff
@@ -42,16 +42,15 @@ - (void)setupWithAppKey:(nonnull NSString *)appKey
     self.appKey = appKey;
     self.appLink = appLink;
     self.titleType = TitleTypeDefault;
+    self.locale = @"en_US";
```

Comment:
should not we update API to set locale explicitly?

Reply:
I don't think so. There will be a setter setLocale. This is optional. Default will always be english

Reply:
OKay, then